src/engine.rs

```diff
@@ -296,7 +296,7 @@ impl LiveEvent {
 /// engine. Changing the default author will not be persisted.
 ///
 /// If set to `Persistent`, the default author will be loaded from and persisted to the specified
-/// path (as base32 encoded string of the author's public key).
+/// path (as hex encoded string of the author's public key).
```

Comment:
This will break existing installations, right? Not sure what the consequence is of the default author being not readable anymore.

Reply:
yeah, the future of this is unclear atm